### PR TITLE
[README] Add Discord server in README

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose

Add link to Discord server in README

JIRA issue: N/A

## Proposed changes
Upadate README.md

## TODO

- [x] Should I add to README Discord server is not fully supported

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@
 [Official chat](https://chat.reactos.org/) &bull;
 [Wiki](https://reactos.org/wiki/) &bull;
 [Forum](https://reactos.org/forum/) &bull;
+[Discord](https://discord.gg/7knjvhT) &bull;
```
[the remark this answers is not in the document]
Comment (Andrej123456789) [reply README.md:35]: `Unofficial Community Discord` is better maybe

File: README.md [hunk 2]
```diff
@@ -32,6 +32,7 @@
 [Official chat](https://chat.reactos.org/) &bull;
 [Wiki](https://reactos.org/wiki/) &bull;
 [Forum](https://reactos.org/forum/) &bull;
+[Unofficial Community Discord](https://discord.gg/7knjvhT) &bull;
```
Comment (binarymaster) [inline README.md:35]: Either just "Community Discord" or "Community-endorsed Discord" please. Personally I'd just prefer the former.

Comment (Andrej123456789) [reply README.md:35]: I will fix it

File: README.md [hunk 3]
```diff
@@ -32,6 +32,7 @@
 [Official chat](https://chat.reactos.org/) &bull;
 [Wiki](https://reactos.org/wiki/) &bull;
 [Forum](https://reactos.org/forum/) &bull;
+Community Discord](https://discord.gg/7knjvhT) &bull;
```
Comment (binarymaster) [inline README.md:35]:
```suggestion
[Community Discord](https://discord.gg/7knjvhT) &bull;
```

**Pull request merged**